### ISSUE TITLE
Cura 4.0 stuck window

### ISSUE THREAD

**Issue by captNtinker** [Ultimaker/Cura#5443]
![Uploading image.png…]()
Project screen loads off my screen, it's anchored to the Right side and will not move ... I can't drag it and when maximized does not let me select the menu items (they son't pull down) program is useless in this manner :(

**Comment by fieldOfView** [comment]
Your screenshot did not get through. Could you try again? Note that you have to visit this issue to add images (https://github.com/Ultimaker/Cura/issues/5443), attaching them through email won't work.

**Comment by fieldOfView** [comment]
I guess you meant this image:
![](https://user-images.githubusercontent.com/47804922/54501429-a4c6e480-48fb-11e9-9dc7-6b519555e593.png)

**Comment by captNtinker** [comment]
Yes, that’s the image!

**Comment by fieldOfView** [comment]
Please continue the discussion here, not in the other issue.

I am afraid your issue is a sideeffect of your screen-resolution; since the redesign the minimum vertical screensize supported seems to be 800 pixels (eg 1280 x 800).

**Comment by Ghostkeeper** [comment]
Yeah. I don't think we'll reduce the minimum screen size any further than what it is in the 2nd beta.

There is a workaround though. First close Cura. Then open the file `C:\Users\<username>\AppData\Roaming\cura\4.0\cura.cfg` with a text editor and change the following preferences:

```
window_left = 0
window_top = 0
```

Setting them both to 0 should cause the window to be aligned to the top left of your screen, causing more of the Cura window to be visible.

**Comment by GregValiant** [comment]
I'm cleaning up these old reports.  I'll go ahead and close this.  If the problem has persisted into 5.13.0 please make out a new report.
